dart/lib/src/logs_enricher_integration.dart

```diff
@@ -0,0 +1,32 @@
+import 'dart:async';
+import 'package:meta/meta.dart';
+
+import 'utils/os_utils.dart';
+import 'integration.dart';
+import 'hub.dart';
+import 'protocol/sentry_log_attribute.dart';
+import 'sentry_options.dart';
+
+@internal
+class LogsEnricherIntegration extends Integration<SentryOptions> {
+  static const integrationName = 'LogsEnricherIntegration';
```

Comment:
```suggestion
  static const integrationName = 'LogsEnricher';
```

let's omit the integration suffix from now on, I'll make sure the other integration names will also be in this format